help/mobile-notifications.md

```diff
@@ -167,6 +167,18 @@ To enable push notifications for your organization:
 
 {end_tabs}
 
+#### Warning banners
+
+To make sure that you are aware when your server does not have mobile
+notifications enabled, you will see a warning banner if:
+
+- Your server's access to mobile push notifications is about to end. This banner
+  is first shown to server administrators, and then to all users.
+- Notifications are not enabled on your server.
```

Comment:
The second bullet might be best expanded to spell out the specific situations, possibly with links, maybe something like this:

- Your server is not [configured to use the mobile push notifications service](https://zulip.readthedocs.io/en/stable/production/mobile-push-notifications.html).
- Your server is configured to use the mobile push notifications service, but is not signed up for on a plan that offers mobile notifications, and either the server has more than 10 users or is not uploading the basic metadata required for the service to confirm it has 10 or fewer users.